src/Backend/SgateShopgatePlugin/Components/Favorites.php

```diff
@@ -141,11 +141,15 @@ public function deleteFromFavoriteList($request)
      */
     private function isWishList()
     {
-        $pluginManager  = $this->container->get('shopware_plugininstaller.plugin_manager');
-        $plugin = $pluginManager->getPluginByName('SwagAdvancedCart');
-        $config = $this->container->get('shopware.plugin.cached_config_reader')->getByPluginName('SwagAdvancedCart');
+        try {
+            $pluginManager  = $this->container->get('shopware_plugininstaller.plugin_manager');
+            $plugin = $pluginManager->getPluginByName('SwagAdvancedCart');
+            $config = $this->container->get('shopware.plugin.cached_config_reader')->getByPluginName('SwagAdvancedCart');
 
-        return $plugin->getInstalled() && $plugin->getActive() && $config['replaceNote'];
+            return $plugin->getInstalled() && $plugin->getActive() && $config['replaceNote'];
+        } catch (\Exception $error) {
+            return false;
```

Comment:
Does it mean that Favorites only work when SwagAdvancedCart is installed? Isn't there a default implementation of Shopware Favorites/Wishlist?